lib/src/entities.ts

```diff
@@ -24,12 +24,12 @@ export class FieldValidationResult {
 
 export class FormValidationResult {
   succeeded: boolean;
-  fieldErrors: FieldValidationResult[];
+  fieldErrors: { [key: string]: FieldValidationResult };
   formGlobalErrors: FieldValidationResult[];
 
   constructor() {
     this.succeeded = false;
-    this.fieldErrors = [];
+    this.fieldErrors = {};
```

Comment:
MMM... I think this is a breaking change, should we go for a major release?

Reply:
Yes, I think it should be major